in_text/text_02_drivers.yml

```
@@ -326,14 +342,14 @@ entities:
         
         nhdhr_120022743 is Cannonsville Reservoir and nhdhr_151957878 is Pepacton Reservoir 
       attr-defs: >- 
-        http://nhd.usgs.gov/
+        https://www.usgs.gov/national-hydrography/nhdplus-high-resolution
       data-min: NA
       data-max: NA
       data-units: NA
     -
       attr-label: site_id
       attr-def: >-
-        Site identifier from the data provider.
+        Site identifier from the data provider. 
```

who is the data provider?
```suggestion
        Site identifier from the data provider.
```